ovh/data_source_ovh_cloud_project_kube.go

```diff
@@ -93,6 +97,7 @@ func dataSourceCloudProjectKubeRead(d *schema.ResourceData, meta interface{}) er
 	d.Set("name", res.Name)
 	d.Set("next_upgrade_versions", res.NextUpgradeVersions)
 	d.Set("nodes_url", res.NodesUrl)
+	d.Set("privateNetworkId", res.PrivateNetworkId)
```

Comment:
should be named private_network_id

Reply:
i've double- checked it, i think you forgot to pass the network id to the POST api call, i'd like to release the provider this morning so i think i'll add it for you if you dont mind